README.md

```diff
@@ -2,4 +2,88 @@
 
 ## hadoop-azure-token-providers
 
-TODO
\ No newline at end of file
+The [`hadoop-azure` library](https://hadoop.apache.org/docs/stable/hadoop-azure/abfs.html) supports supplying a [custom token provider](https://hadoop.apache.org/docs/stable/hadoop-azure/abfs.html#Custom_OAuth_2.0_Token_Provider) for getting accessing tokens for Azure Storage.
```

Comment:
All of Azure Storage or a specific service via a specific protocol?

Reply:
You're right, it's just via `abfs(s)` protocol